Automattic/Gravatar-SDK-Android
# Fix: Fix blank space in LargeProfile without description

Closes #210 

### Description

Same as with the `Profile` component the `LargeProfile` has an extra amount of empty space when the profile has no description. This PR fixes that.

Here's the screenshot comparison:
<img width="2533" alt="Screenshot 2024-07-04 at 14 37 51" src="https://github.com/Automattic/Gravatar-SDK-Android/assets/12660801/8c47d842-bac0-47aa-8cba-90bc7210ea6c">


On top of that, now that we have the extra `skeletonModifier` I was able to modify the previously added check in the `Profile` component. There's no need to do an extra if check. There are no UI changes - tests are passing. 

### Testing Steps

1. Remove description from your Gravatar profile
2. Open the demo app
3. Go to the profile tab
4. Fetch the profile
5. Confirm there's less blank space without description
6. Confirm the skeleton still looks ok and has proper spacing
